docs/src/pages/components/icons/icons.md

```diff
@@ -80,13 +41,38 @@ The safest is option 1 but option 2 can yield the best developer experience.
 Make sure you follow the [minimizing bundle size guide](/guides/minimizing-bundle-size/#option-2) before using the second approach.
 The configuration of a Babel plugin is encouraged.
 
-### More SVG icons
+Each icon also has a "theme": `Filled` (default), `Outlined`, `Rounded`, `Two tone` and `Sharp`. If you want to import the icon component with a "theme" different than default, append the "theme" name to the icon name. For example `@material-ui/icons/Delete` icon with:
+
+- `Filled` "theme" (default) is exposed as `@material-ui/icons/Delete`,
+- `Outlined` "theme" is exposed as `@material-ui/icons/DeleteOutlined`,
+- `Rounded` "theme" is exposed as `@material-ui/icons/DeleteRounded`,
+- `Two tone` "theme" is exposed as `@material-ui/icons/DeleteTwoTone`,
+- `Sharp` "theme" is exposed as `@material-ui/icons/DeleteSharp`.
```

Comment:
Is there a better word than "exposed"?

Reply:
exposed was used in the previous version of the document. I wasn't sure if it is good, but I tried to use "exported" in other places. I think I missed here. Do you agree "exported" is good?

Reply:
Probably not technically correct, but I think it's better than "exposed".